internal/commands/build/build.go

```diff
@@ -45,8 +45,8 @@ type buildOptions struct {
 	noResolveImage bool
 }
 
-const buildExample = `- $ docker app run --name myrunningapp myrepo/myapp:mytag
-- $ docker app run 34be4a0c5f50 --name myrunningapp`
+const buildExample = `- $ docker app build .
+- $ docker app build . -f myapp.dockerapp -t myrepo/myapp:1.0.0`
```

Comment:
```suggestion
- $ docker app build -f myapp.dockerapp -t myrepo/myapp:1.0.0 .`
```

Just to respect the order in usage.